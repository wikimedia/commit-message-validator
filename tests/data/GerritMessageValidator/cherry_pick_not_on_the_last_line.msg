This is a commit subject

This is a commit body

(cherry picked from commit a24ca11e277afd8b5259d44b2d645d4dbb99502f)
Change-Id: If89d24838e326fe25fe867d02181eebcfbb0e196
