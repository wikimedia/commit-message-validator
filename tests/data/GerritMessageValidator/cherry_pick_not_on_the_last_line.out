The following errors were found:
- Line 5: Cherry pick line is not the last line
